_data/projects/InstascanLibrary.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: InstascanLibrary
+desc: It is about how you can easily scan QR code using web technologies.
+site: https://github.com/schmich/instascan
+tags:
+- QR code
+- Instascan
+- JavaScript
+- JQuery
+- async
+- sql
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/schmich/instascan/labels
```

Comment:
@PrajapatiAnkit Please use a label which is already there such as `help wanted` or create a new one. Change the name and link accordingly. Link should be like https://github.com/schmich/instascan/labels/help%20wanted